datahub-web-react/src/app/ingest/source/builder/LookerWarning.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Alert } from 'antd';
+import { LOOKER, LOOK_ML } from './constants';
+
+const LOOKML_DOC_LINK = 'https://datahubproject.io/docs/generated/ingestion/sources/looker#module-lookml';
+const LOOKER_DOC_LINK = 'https://datahubproject.io/docs/generated/ingestion/sources/looker#module-looker';
+
+interface Props {
+    type: string;
+}
+
+export const LookerWarning = ({ type }: Props) => {
+    let link: React.ReactNode;
+    if (type === LOOKER) {
+        link = (
+            <a href={LOOKER_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub looker module
+            </a>
+        );
+    } else if (type === LOOK_ML) {
+        link = (
+            <a href={LOOKML_DOC_LINK} target="_blank" rel="noopener noreferrer">
+                DataHub lookml module
+            </a>
+        );
+    }
```

Comment:
can you swap these two links? If they're on the Looker form, we need to tell them to also use the LookML module. Then if they're on the LookML form, we need to tell them to use the Looker module.

Reply:
Checked and fixed it!!